src/components/nav-links.js

```diff
@@ -0,0 +1,35 @@
+import MenuIcon1 from "../assets/menu-icons-02.png";
+import MenuIcon2 from "../assets/menu-icons-03.png";
+import MenuIcon3 from "../assets/menu-icons-01.png";
+import MenuIcon4 from "../assets/menu-icons-04.png";
+
+const NavLinks = () => {
+  return (
+    <>
+      <div className="button-1-wrapper">
+        <button className="button-1">
+          <a href="#basics-section">
+            <img src={MenuIcon1} alt="" />
+          </a>
+        </button>
+        <button className="button-1">
+          <a href="#intervals-section">
+            <img src={MenuIcon2} alt="" />
+          </a>
+        </button>
+        <button className="button-1">
+          <a href="#major-scales-section">
+            <img src={MenuIcon3} alt="" />
+          </a>
+        </button>
+        <button className="button-1">
+          <a href="#circle-of-fifths-section">
+            <img src={MenuIcon4} alt="" />
+          </a>
+        </button>
```

Comment:
This here seems quite repetitive. We could probably use an array of objects for this, and render this via a array.map!

Reply:
```suggestion
        const buttons = [{ href: "sds", icon: <MenuIcon1 /> }, .....]
        
        buttons.map((buttonItem) => 
            <button className="button-1">
          <a href={buttonItem.href}>
            <img src={buttonItem.icon} alt="" />
          </a>
        </button>
        )
```